packages/design-system-docs/src/pages/components/Button/button.example.html

```diff
@@ -32,7 +34,44 @@ <h6 class="preview__label">Primary button</h6>
   Disabled
 </button>
 
+<h6 class="preview__label">Secondary button</h6>
+<p><code>.ds-c-button .ds-c-button--secondary</code></p>
+<button type="button" class="ds-c-button ds-c-button--secondary">
+  Default
+</button>
+<button type="button" class="ds-c-button ds-c-button--secondary ds-c-button--hover">
+  Hover
+</button>
+<button type="button" class="ds-c-button ds-c-button--secondary ds-c-button--active">
+  Active
+</button>
+<button type="button" class="ds-c-button ds-c-button--secondary ds-c-button--focus">
+  Focus
+</button>
```

Comment:
The test on HCgov design system has a grayed out Secondary Focus button. Not an issue here, just want to note it here as we may need to add an override over at the HCgov ds

Reply:
Hmm, I'll look back into that

Reply:
I'm seeing this on HCgov DS
![Screen Shot 2021-05-26 at 1 24 12 PM](https://user-images.githubusercontent.com/20322880/119704652-e5004680-be25-11eb-9ec3-7cd1404e8cd9.png)

Reply:
That's probably because we would need to have unique theming for HCgov and Mgov